src/test/scenarios.node.ts

```diff
@@ -0,0 +1,63 @@
+import { TransportLocalScenario } from './scenarios.universal.ts';
+import { FnsBag } from '../types-bag.ts';
+import { IConnection, ITransport } from '../types.ts';
+import { ITransportScenario } from './scenario-types.ts';
+import { TransportHttpClient } from '../transport-http-client.ts';
+import { TransportHttpServerExpress } from '../transport-http-server-express.ts';
+import { default as express, Express } from 'https://esm.sh/express@4.17.2?dts';
+import { sleep } from '../util.ts';
+
+class TransportHttpExpressScenario<BagType extends FnsBag> implements ITransportScenario<BagType> {
+    name = 'TransportHttpClient + TransportHttpServerOpine';
+    connAtoB: IConnection<BagType>;
+    connBtoA: IConnection<BagType>;
```

Comment:
could this be `IConnection<BagType> | null` so we don't have to do the convoluted typescript dance on line 31?

Reply:
This will mean that everywhere else will need to do null-checking for connBtoA. Acceptable?